test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -37,8 +37,8 @@
 #include "timestamp_manager.h"
 
 session_simulator::session_simulator()
-    : _has_commit_ts(false), _ts_round_read(false), _txn_running(false), _commit_ts(0),
-      _durable_ts(0), _first_commit_ts(0), _prepare_ts(0), _read_ts(0)
+    : _has_commit_ts(false), _ts_round_prepared(false), _ts_round_read(false), _txn_running(false),
```

Comment:
The list starts to be big, next time we may want to consider having an `init()` function that would init everything.